.github/workflows/ci.yml

```diff
@@ -11,28 +11,45 @@ jobs:
 
   test:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     continue-on-error: ${{ matrix.experimental }}
 
     strategy:
       matrix:
         include:
           - mediawiki_version: '1.35'
-            chameleon_version: '4.1.0'
+            smw_version: '4.2.0'

```

Comment:
does the skin require SMW?

Reply:
I will check and remove it if not required.